src/valeriy/week1/FINRA.java

```diff
@@ -1,4 +1,32 @@
 package valeriy.week1;
 
 public class FINRA {
+
+    public static void main(String[] args) {
+        fINRA(30);
+    }
+
+
+
+    public static void fINRA(int num) {
+        for (int i = 1; i <= num; i++) {
+            if (i % 5 == 0 && i % 3 == 0) {
+                System.out.print(" FINRA ");
+            } else if (i % 5 == 0) {
+                System.out.print(" RA ");
+            } else if (i % 3 == 0) {
+                System.out.print(" FIN ");
+            } else {
+                System.out.print(i+" ");
+            }
+        }
+    }
+
 }
+  /*
```

Comment:
Looks perfect to me

Reply:
awww you are too kind 
now i need you (it is in a menu "review changes" ) or Ideen to accept my pull request request please :)